entgql/annotation.go

```diff
@@ -35,6 +35,16 @@ type Annotation struct {
 	Type string `json:"Type,omitempty"`
 	// Skip exclude the type
 	Skip bool `json:"Skip,omitempty"`
+	// Description is description of the GQL field/type.
+	Description string `json:"Description,omitempty"`
+	// Directives to add on the field/type.
+	Directives []Directive `json:"Directives,omitempty"`
+	// RelayConnection expose this node as a relay connection.
+	RelayConnection bool `json:"RelayConnection,omitempty"`
```

Comment:
it might make sense to expand this into a more full config object.

E.g.

```go
type RelayConnectionConfig struct {
  Enabled bool
  IncludeTotal bool
  EdgeNotNull bool
  NodeNotNull bool
}
```

Reply:
`Edge` and `Node` nullability was defined in the relay spec. I don't think we need options for them.

Reply:
but there is no way for the user to control them - but this might be ok.